pkg/virt-controller/watch/network_test.go

```diff
@@ -261,6 +262,83 @@ var _ = Describe("Network interface hot{un}plug", func() {
 			[]v1.VirtualMachineInterfaceRequest{},
 		),
 	)
+
+	Context("handle dynamic interface requests", func() {
+		var clusterConfig *virtconfig.ClusterConfig
+
+		const vmiHasPodNetwork = true
+
+		BeforeEach(func() {
+			clusterConfig, _, _ = testutils.NewFakeClusterConfigUsingKVConfig(&v1.KubeVirtConfiguration{})
+		})
+
+		DescribeTable("for a VM with no interfaces",
+			func(requests []v1.VirtualMachineInterfaceRequest, expectedInterfaces []v1.Interface, isPodNetworkOnVmi bool) {
+				vm := kubecli.NewMinimalVM("test")
+				vm.Spec.Template = &v1.VirtualMachineInstanceTemplateSpec{}
+				vm.Status.InterfaceRequests = requests
+
+				Expect(handleDynamicInterfaceRequests(vm, isPodNetworkOnVmi, clusterConfig)).To(Succeed())
+				Expect(vm.Spec.Template.Spec.Domain.Devices.Interfaces).To(Equal(expectedInterfaces))
+			},
+			Entry("no request",
+				[]v1.VirtualMachineInterfaceRequest{},
+				nil,
+				vmiHasPodNetwork,
+			),
+			Entry("one hot plug request",
```

Comment:
nit: It would be nice to hint what expected to happen.

Reply:
done.